hyp3_rtc_gamma/__main__.py

```diff
@@ -214,6 +212,16 @@ def find_product(dir_):
     return None
 
 
+def find_product_name(dir_):
+    files = glob.glob(f'{dir_}/**')
+    readme_file = None
+    for file in files:
+        if file.endswith('.README.txt'):
+            readme_file = file
+            break
+    return readme_file.split('.')[0]
```

Comment:
This implementation is pretty lazy and doesn't have tests.  Could refactor all 5 `find_...` functions, they're mostly copy/paste of each other, but I'm not feeling up to it right now.

Reply:
We'll do some cleanup and moving stuff into hyp3lib as we start adding more processes, so we can circle back to this then